submissions/AbrahamJudah/HTML&CSS/VivaMart.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>VivaMart.....An Online store for all your quality products</title>
+    <link rel="stylesheet" type="text/css" href="styler.css">
+    <script src="https://kit.fontawesome.com/a126fa987a.js"></script>
+</head>
+<body>
+    <div class="container">
+        <section class="subhead">
+            <div class="header">
+                <img src="Pics/star.png" alt="star" width="30" height="30">
+                <p class="note">Welcome!</p>
+                <a href="#">Join Free</a>&nbsp;<span>or</span>&nbsp; <a href="#">Sign In</a>
+            </div>
+            <div class="header1">
+                <ul>
+                    <li><a href="#" target=_blank>Help <i class="fas fa-angle-down"></i></a></li>
+                    <li><a href="#" target="_blank">Use App!</a></li>
+                    <li><a href="#" target="_blank"><i class="far fa-heart"></i> Wishlist</a></li>
+                </ul>
+        </section>
+        <header>
+            <img src="Pics/logo.png" alt="logo" width="140" height="60">
+            <div class="searchbox">
+                <form action="/action_page.php">
+                <input type="text" placeholder="Search.." name="search">
+                <button type="submit"><i class="fas fa-search fa-2x"></i></button>
+                </form>
+            </div>
+            <div class="signin">
+                <ul>
+                    <li><i class="far fa-user fa-2x"></i><a href="#"  class="link1">My Account<i class="fas fa-angle-down"></i></a></li>
+                    <li><a href="#"><i class="fas fa-history fa-2x"></i></a></li>
+                    <li><i class="fas fa-shopping-cart fa-2x"></i><a href="#" target="_blank" class="link2">Cart<i class="fas fa-angle-down"></i></a></li>
+                </ul>
+            </div>
+        </header>
+        <main>
+            <div class="nav">
+                <i class="fas fa-list "></i> &nbsp;Categories
```

Comment:
I see a lot of FontAwesome icon usage in your code but I don't see the icons on the page. Can you make them show?